melos.yaml

```diff
@@ -26,6 +26,12 @@ scripts:
   test:unit:ios: >
     ./.circleci/test_all_plugins.sh ios-test && exit 0
 
+  coverage:ios: >
+    ./build-support/codecov.sh -F ios-unit-tests
+
+  coverage:flutter: >
+    ./build-support/codecov.sh -F flutter-unit-tests
+
```

Comment:
Why not android?

Reply:
This PR is still in progress, was supposed to be only for iOS, but then I got greedy and started adding other 🤷